os_installer2/diskops.py

```diff
@@ -801,18 +814,18 @@ class DiskOpUseHome(BaseDiskOp):
     """ Use an existing home paritition """
 
     home_part = None
-    home_part_fs = None
+    part_type = None
```

Comment:
huh?

Reply:
Just for consistency. I saw everything was format_type, while it seem'd home_part_fs was a little out of place. Totally for my ocd, not a technical need. I'll make sure I don't do that in the future.